pluginpal/strapi-webtools
# Feature/strapi 5

### What does it do?

- Migrate the packages to Strapi 5.
- Rename the packages to `strapi-plugin-x`

### Why is it needed?

For Strapi 5 support.

### How to test it?

- Make sure all tests are successful
- Fetch this branch, follow the Contribution Guidelines and test manually
